.github/workflows/documentation-review-oauth.yml

```
@@ -0,0 +1,532 @@
+              - 'internal/**/*.go'
+              - 'pkg/**/*.go'
+              - 'cmd/**/*.go'
+              - 'scripts/**/*.py'
+
+      - name: Generate documentation matrix
+        if: steps.skip-check.outputs.should_skip != 'true' && steps.changes.outputs.docs == 'true'
+        id: matrix
+        run: |
+          # Group files for parallel processing
+          CHANGED_FILES=$(git diff --name-only HEAD~1 HEAD | grep -E '\.(md)$' || echo "")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)